Lib/multiprocessing/forkserver.py

```diff
@@ -25,6 +26,7 @@
 
 MAXFDS_TO_SEND = 256
 SIGNED_STRUCT = struct.Struct('q')     # large enough for pid_t
+_authkey_len = 32  # <= PIPEBUF so it fits a single write to an empty pipe.
```

Comment:
Should we test this constraint anywhere?

Reply:
I don't think we need to. IIRC, PIPEBUF is defined as ">= 512" by POSIX and it's hard to fathom why it'd _ever_ be less.